<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgIperf</class>
 <widget class="QDialog" name="DlgIperf">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Iperf Pair config</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../qiperf.qrc">
    <normaloff>:/pair-add</normaloff>:/pair-add</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="7" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>window size(-w):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="sb_omit">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Test Duration(-t, sec):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_34">
              <property name="text">
               <string>iperf version:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_version">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_35">
              <property name="text">
               <string>Port(-p, --port)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sb_port">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>5201</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_36">
              <property name="text">
               <string>protocal:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_protocal">
              <item>
               <property name="text">
                <string extracomment="Transmission Control Protocol">TCP</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string extracomment="User Datagram Protocol">UDP</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string extracomment="Stream Control Transmission Protocol">SCTP</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="sb_bitrate">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(0 for unlimited)&lt;/p&gt;&lt;p&gt; (default 1 Mbit/sec for UDP, unlimited for TCP)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="13" column="3">
          <widget class="QComboBox" name="cb_fmtreport">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>k</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>g</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>t</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>K</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>G</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>T</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_25">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;IPv4: 536 octets (= 576 - 20 - 20) &lt;/p&gt;&lt;p&gt;IPv6: 1220 octets (= 1280 - 40 - 20). &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Max segment size(-M,--set-mss):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>bitrate(-b):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>IP DSCP bits(--dscp):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="sb_parallel">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QSpinBox" name="sb_mss">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0: default value&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>8000</number>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Type Of Service(-S, --tos N)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QSpinBox" name="sb_dscp">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-1: default value&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>-1</number>
           </property>
           <property name="maximum">
            <number>63</number>
           </property>
           <property name="value">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="gb_control_server">
              <property name="title">
               <string>Control server</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1">
               <property name="spacing">
                <number>2</number>
               </property>
               <property name="leftMargin">
                <number>2</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="rightMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QLabel" name="label_23">
                 <property name="text">
                  <string>manager server ip:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cb_mserver_ip">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_37">
                 <property name="text">
                  <string>bind to  ip:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cb_server_bind_ip">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string>0.0.0.0</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,1">
               <property name="leftMargin">
                <number>6</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="rightMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <property name="horizontalSpacing">
                <number>2</number>
               </property>
               <property name="verticalSpacing">
                <number>4</number>
               </property>
               <item row="0" column="3">
                <widget class="QLineEdit" name="le_target_ip"/>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="label_32">
                 <property name="text">
                  <string>client bind to ip(-B):</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_31">
                 <property name="text">
                  <string>target server ip(-c):</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QComboBox" name="cb_client_bind_ip">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_24">
                 <property name="text">
                  <string>manager client ip:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="cb_mclient_ip">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>omit(-O, sec):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSpinBox" name="sb_windowsize">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="sb_duration">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0: run forever&lt;/p&gt;&lt;p&gt;default: 10 sec&lt;/p&gt;&lt;p&gt;Max: 864000 sec&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>864000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Reports interval(-i, Sec)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>window size unit:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="3">
          <widget class="QSpinBox" name="sb_tos">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>-1</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="chk_reverse">
              <property name="toolTip">
               <string>Server send data to client</string>
              </property>
              <property name="text">
               <string>reverse(-R)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chk_bidir">
              <property name="text">
               <string>bidirectional(--bidir)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="13" column="2">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>format to report(-f):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QComboBox" name="cb_unit_bitrate">
           <item>
            <property name="text">
             <string>K</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>G</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>bitrate unit:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>Parallel(-P):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QComboBox" name="cb_unit_windowsize">
           <property name="currentText">
            <string>K</string>
           </property>
           <item>
            <property name="text">
             <string>K</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>G</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QSpinBox" name="sb_interval">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Buffer Length(-l):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Buffer Length unit:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="sb_buffer">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QComboBox" name="cb_unit_buffer">
           <item>
            <property name="text">
             <string>K</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>G</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>k</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>m</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tw_iperf">
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>data</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qiperf.qrc"/>
 </resources>
 <connections/>
</ui>
